package.json

```diff
@@ -24,7 +24,6 @@
   "peerDependencies": {
     "code-push": ">=1.8.0-beta",
     "cordova-plugin-file-transfer": ">=1.3.0",
-    "cordova-plugin-file": ">=3.0.0",
```

Comment:
Hey Daniel, we looked into this and it looks like the reason why this is broken is that their [package.json](https://github.com/apache/cordova-plugin-file-transfer/blob/1.5.x/package.json#L48) does not match their plugin.xml, leading to npm giving an error. They appear to have fixed it on their current dev branch but it's not on npm.

Do you even need to have the cordova plugins in package.json as peer dependencies in order to build? Aren't they only needed when cordova installs the plugin?